sympy/polys/polytools.py

```diff
@@ -5202,6 +5203,19 @@ def try_non_polynomial_gcd(seq):
 
     try:
         polys, opt = parallel_poly_from_expr(seq, *gens, **args)
+
+        #for gcd in domain Q[irrational]
+        genrs = opt['gens']
+        if len(genrs) == 1:
+            genr = genrs[0]
+            if not genr.has(Symbol) and genr.is_irrational:
+                if all((element/genr).is_rational for element in seq):
+                    num, den = 0, 1
```

Comment:
Would it be possible to use the `gcd` function for this computation?

Reply:
Yes, I think that would be possible, do you mean replacing igcd with gcd, or using gcd_list and lcm_list for numerators and denominators?